docs_src/microservices/security/Ch-APIGateway.md

```diff
@@ -21,37 +21,37 @@ OAuth2 authentication and ACL.
 ## Start the API Gateway
 
 Start the API gateway with Docker Compose and a Docker Compose manifest
-file (the Docker Compose file named docker-compose-nexus-{redis,mongo}.yml (or -arm64 variabnts) found at
-<https://github.com/edgexfoundry/developer-scripts/tree/master/releases/geneva/compose-files>)).
+file (the Docker Compose file named `docker-compose-ireland.yml` (or -arm64 variabnts) found at
+<https://github.com/edgexfoundry/developer-scripts/tree/master/releases/ireland/compose-files>)).
```

Comment:
this has to be updated since we've moved the compose to new repo `edgex-compose`